### ISSUE TITLE
LC0003, LC0005 false positives raised for dictionaries of interfaces

### ISSUE THREAD

**Issue by MODUSCarstenScholling** [StefanMaron/BusinessCentral.LinterCop#1008]
# Description
LC0003 and LC0005 false positives are raised for new syntax [Create lists and dictionaries of interfaces in AL](https://learn.microsoft.com/en-us/dynamics365/release-plan/2025wave1/smb/dynamics365-business-central/create-list-dictionary-interfaces-al).

## Versions
- BusinessCentralLinterCop.dll: 0.35.1.0
- ms-dynamics-smb.al: 16.0.1384372

## Example

```AL
    internal procedure InterfacesDictionaryAndList()
    var
        dictAIServiceName: Dictionary of [Integer, Interface "AI Service Name"];
        listAIServiceName: List of [Interface "AI Service Name"];
    begin
        Clear(dictAIServiceName);
        Clear(listAIServiceName);
        // Causes the following warnings for the phrase "AI Service Name" in dictAIServiceName:
        // LC0003: Do not use an Object ID for properties or variables declaration. Use Dictionary instead.
        // LC0005: Wrong casing detected! Use Dictionary instead.
        // List has no issues afaik.
    end;
```

**Comment by Arthurvdv** [comment]
Hi, over the past months I’ve been putting a lot of time and care into the successor of BusinessCentral.LinterCop: ALCops (https://alcops.dev/). Because of that, this repository is no longer actively maintained.

I’m closing this issue, as it won’t be handled here anymore. If this is still relevant for you, please have a look at ALCops/Analyzers (https://github.com/ALCops/Analyzers).

To make the transition easier, you can also follow the LinterCop Migration Guide: https://alcops.dev/docs/lintercop-migration/

If you have any questions or run into issues during the transition, feel free to reach out through ALCops. Happy to help 🙂
